src/ifcopenshell-python/ifcopenshell/ids.py

```diff
@@ -154,10 +156,10 @@ def to_xml(self, filepath="./", ids_schema=ids_schema):
             },
         )
 
-        with open(filepath, "w") as f:
-            f.write('<?xml version="1.0" encoding="UTF-8"?>\n')
-            f.write("<!-- IDS (INFORMATION DELIVERY SPECIFICATION) CREATED USING IFCOPENSHELL -->\n")
-            f.write(ids_str)
+        with open(filepath, "wb") as f:
```

Comment:
why this change? Did you consider:

    open(filepath, "w", encoding="utf-8")

and don't do the manual encodes?

Reply:
you're right, the use of binary mode is not needed here. Adjusted accordingly in 0b82381.